app/views/core_induction_programmes/mentor_materials/show.html.erb

```diff
@@ -5,6 +5,6 @@
     <% end %>
     <h1 class="govuk-heading-xl"><%= @mentor_material.title %></h1>
 
-    <%= render GovspeakComponent.new(content: @mentor_material.content) %>
+    <%= render GovspeakComponent.new(content: @mentor_material.mentor_material_parts_in_order[0].content) %>
```

Comment:
Let's just remove it, if there are any mentor material parts we will get redirected to the part page.

Reply:
oh yea this was here for debugging!